coins/src/adapters/index.ts

```diff
@@ -174,4 +174,7 @@ export default {
   goblin: require("./markets/goblin"),
   ember: require("./yield/ember"),
   suirewards: require("./markets/suirewards"),
+  axlp: require("./liquidStaking/axlp"),
+  morphoBlue: require("./moneyMarkets/morpho"),
+  lista: require("./moneyMarkets/morpho"),
```

Comment:
if morphoBlue has multiple adapters, better to do something like what we have for euler: 
<img width="530" height="207" alt="image" src="https://github.com/user-attachments/assets/007b17c0-1d2f-4e7c-a853-d93f53393d57" />